ops/mappings.json

```diff
@@ -2,623 +2,56 @@
-        "mapping": "UGenericScheme",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "SchemeRegistrar",
-        "contractName": "SchemeRegistrar",
-        "dao": "base",
-        "mapping": "SchemeRegistrar",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "UController",
-        "contractName": "UController",
-        "dao": "base",
-        "mapping": "UController",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "DAORegistry",
-        "contractName": "DAORegistry",
-        "dao": "base",
-        "mapping": "DAORegistry",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "Avatar",
-        "contractName": "Avatar",
-        "dao": "dao",
-        "mapping": "Avatar",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "DAOToken",
-        "contractName": "DAOToken",
-        "dao": "dao",
-        "mapping": "DAOToken",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "Reputation",
-        "contractName": "Reputation",
-        "dao": "dao",
-        "mapping": "Reputation",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "Controller",
-        "contractName": "Controller",
-        "dao": "dao",
-        "mapping": "Controller",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "Avatar",
-        "contractName": "Avatar",
-        "dao": "test",
-        "mapping": "Avatar",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "DAOToken",
-        "contractName": "DAOToken",
-        "dao": "test",
-        "mapping": "DAOToken",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "Reputation",
-        "contractName": "Reputation",
-        "dao": "test",
-        "mapping": "Reputation",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "Reputation",
-        "contractName": "DemoReputation",
-        "dao": "organs",
-        "mapping": "Reputation",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "DAOToken",
-        "contractName": "DemoDAOToken",
-        "dao": "organs",
-        "mapping": "DAOToken",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "Avatar",
-        "contractName": "DemoAvatar",
-        "dao": "organs",
-        "mapping": "Avatar",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "Redeemer",
-        "contractName": "Redeemer",
-        "dao": "base",
-        "mapping": "Redeemer",
-        "arcVersion": "0.0.1-rc.33"
-      },
-      {
-        "name": "DAOTracker",
-        "contractName": "DAOTracker",
-        "dao": "base",
-        "mapping": "DAOTracker",
-        "arcVersion": "0.0.1-rc.33"
-      }
-    ]
+        {
+          "name": "GEN",
+          "contractName": "GEN",
+          "dao": "package",
+          "mapping": "DAOToken",
+          "arcVersion": "0.0.1-rc.2"
```

Comment:
this arc version might be confusing with the legacy one .
maybe we should give it other name ? 
`1.0.1-rc.2` or `exp-0.0.1-rc.2`?

Reply:
Ok, shouldn't be a problem, but I'll need you to first rename the arc package like this.